integrations/access/accessrequest/app.go

```diff
@@ -120,6 +138,19 @@ func (a *App) run(ctx context.Context) error {
 		return trace.Wrap(err)
 	}
 
+	amrs, err := a.getAllAccessMonitoringRules(ctx)
```

Comment:
```suggestion
	rules, err := a.getAllAccessMonitoringRules(ctx)
```
Nit: call these rules. "AMR" is not something most Teleport devs are going to recognize.